splash/src/main/kotlin/edu/artic/splash/SplashActivity.kt

```diff
@@ -9,4 +12,17 @@ class SplashActivity : BaseViewModelActivity<SplashViewModel>() {
 
     override val viewModelClass: KClass<SplashViewModel>
         get() = SplashViewModel::class
+
+    override fun onViewModelCreated(viewModel: SplashViewModel) {
+        super.onViewModelCreated(viewModel)
+        val handler = Handler()
+
+        /**
+         * TODO:: Remove the post delay once splash is functional.
+         */
+        handler.postDelayed({
```

Comment:
I like the TODO. ok. Generally should provide a mechanism to cancel the callback if someone leaves the screen so that the app wont launch on top of another app, if they say left the app. This is fine for now.